app/store/appStore.js

```diff
@@ -6,10 +6,11 @@ import rootReducer from '../reducers';
 export default function configureStore(initialState) {
   const store = createStore(rootReducer,
     initialState,
-    applyMiddleware(thunk),
     compose(
-    window.devToolsExtension ? window.devToolsExtension() : _ => _
-  ));
+      applyMiddleware(thunk),
+      window.devToolsExtension ? window.devToolsExtension() : _ => _
```

Comment:
good catch. can you change the `_` to an `f` while you're at it to make it a bit clearer? thanks.